bundle/phases/bind.go

```diff
@@ -24,7 +24,6 @@ func Bind(ctx context.Context, b *bundle.Bundle, opts *terraform.BindOptions) {
 	}()
 
 	bundle.ApplySeqContext(ctx, b,
-		statemgmt.StatePull(),
```

Comment:
Why don't we pull state anymore?

Reply:
we do, it's a function called PullResourcesState()